azure-pipelines.yml

```diff
@@ -65,6 +65,32 @@ jobs:
             displayName: "npm build and test"
           - script: npm run test:nolint
             displayName: "npm build and test (no linting)"
+    - job: SmokeTest

```

Comment:
I would prefer to include this smoke test in the above "UnitTests" job (You can rename it to just "Test" if you want). Then you get the benefit of testing the whole matrix and it simplifies the yaml because it gets rid of some duplicate steps

Reply:
Done